base/_netnode.py

```diff
@@ -116,6 +116,12 @@ class netnode(object):
         altfirst = _ida_netnode.netnode_altfirst
         altnext = _ida_netnode.netnode_altnext
 
+    # default tags
+    alttag = idaapi.atag
+    suptag = idaapi.stag
+    hashtag = idaapi.htag
+    chartag = 0x64
```

Comment:
I think on older versions of IDA, this can't be an integer and must be a char/byte.

Reply:
Probably worth adding a comment describing where I reversed this value from.